package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "legalize",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Legalize follows semver:

    Bug fixes not affecting the API increment the patch version

which seems to be the case.